DrinkACupOfJuice/html/jsn/answern.js

```
@@ -0,0 +1,115 @@
+        return response.json();
+    })
+    .then(data => {
+        if (!Array.isArray(data)) {
+            throw new Error('Data received is not an array');
+        }
+
+        const container = document.getElementById('questionsAndAnswers');
+        container.innerHTML = ''; // 清空容器内容
+
+        data.forEach(questionObj => {
+            if (questionObj.hasOwnProperty('id') && questionObj.hasOwnProperty('title') && Array.isArray(questionObj.answers)) {
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)